### ISSUE TITLE
Better UI

### ISSUE THREAD

**Issue by redhog** [geowurster/pyin#17]
Here are some suggestions for better UI:

Remove --write-true and --on-true options and provide the following:

```
pyin MAP_EXPR
pyin FILTER_EXPR MAP_EXPR
```

The latter is equivalent to the current expression:

```
pyin --write-true --on-true=MAP_EXPR FILTER_EXPR
```

Change --reader / --writer to take an expression (that has a infile/outfile object available in a variable). Example usage:

```
pyin --reader=csv.DictReader(infile)
```

This removes the need for --reader-option and --writer-option

I'm not sure the -v option is needed:

```
pyin -v 'foo=bar'
```

is equivalent to

```
pyin -s 'foo="bar"'
```
- all you win is some quoting...

Same goes for -im:

```
pyin -im foo.bar.fie
```

is quivalent to

```
pyin -s 'import foo.bar.fie'
```

In short: Less options and more pure python would make the UI easier to learn!
